cip/2.testable/CIP2015-06-24-call-procedures.adoc

```diff
@@ -102,65 +128,40 @@ This calls `myProc` for each record produced by the preceding `MATCH` clause wit
 Each call may yield multiple procedure result records with a single result field `last`.
 For each of them, a new statement result record is created that contains the original variables already in scope (i.e. `n`, `r`, `g`) as well as the projected result field `last` renamed as `lastLogin`.
 
+Omitting the `YIELD` subclause means that no new variables are introduced into the scope.
+The procedure call still has the same effect on the cardinality of the query execution in this case.
```

Comment:
Oh - I think I was typing when you made your suggestion as I only saw it afterwards...
Your suggestion is a massive improvement both on the original and on my suggested amendment +10